src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -166,17 +168,32 @@ public int getTotalMessageCount() {
 
   }
 
-
-
-  /* About me Section */
   /** Stores the User in Datastore. */
   public void storeUser(User user) {
     Entity userEntity = new Entity("User", user.getEmail());
     userEntity.setProperty("email", user.getEmail());
-    userEntity.setProperty("aboutMe", user.getAboutMe());
```

Comment:
why not have an aboutme?

Reply:
Profile is about me essentially except extended, so that specific information is requested. I will just change it to include a bio but for now alan's branch should have about me so don't fret. Also, this is a design question and we discussed this I believe.

Reply:
Also a design question is your question about profiles. Profiles and users are not the same. Each User HAS-A Profile; Profile is a sub-entity of User. Very similar to each User having an About-me. Literally the same except now About-me is not just a bio. It is your name, bio, etc.

Reply:
Fair enough. Thanks!